src/main/java/stream/flarebot/flarebot/commands/moderation/mod/UnmuteCommand.java

```diff
@@ -0,0 +1,60 @@
+package stream.flarebot.flarebot.commands.moderation.mod;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.util.GeneralUtils;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.awt.Color;
+
+public class UnmuteCommand implements Command {
+
+    @Override
+    public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message message, String[] args, Member member) {
+        if (args.length != 1) {
+            MessageUtils.getUsage(this, channel, sender).queue();
+        } else {
+            User user = GeneralUtils.getUser(args[0], guild.getGuildId());
+            if (user == null) {
+                MessageUtils.sendErrorMessage("Invalid user!!", channel);
+                return;
+            }
+            if (guild.getGuild().getMember(user).getRoles().contains(guild.getMutedRole())) {
```

Comment:
Check if getMutedRole is null

Reply:
Unless binary changed muted role creation it check null on getMutedRole

Reply:
Okie, what do I do if it is?

Reply:
I'm guessing do nothing

Reply:
@binaryoverload you changed muted role creation?

Reply:
No he just moved it.